easybuild/easyconfigs/m/minieigen/minieigen-0.5.3-Python-2.7.10.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
```

Comment:
The name of the file is incomplete (toolchain and toolchainversion are missing).